android/hybridcommon-ui/src/main/java/com/revenuecat/purchases/hybridcommon/ui/PaywallFragment.kt

```diff
@@ -94,6 +96,14 @@ internal class PaywallFragment : Fragment(), PaywallResultHandler {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        // This should normally never happen, but just in case, we don't want to try to present the paywall
+        // if the SDK is not configured.
+        if (!Purchases.isConfigured) {
```

Comment:
I don't think this is needed... But I added it as a failsafe, in case we're missing any more cases.